scripts/editor/output-css-variables.js

```diff
@@ -97,20 +97,40 @@ export const outputCssVariables = (attributes, manifest, unique) => {
 			continue;
 		}
 
+		if (value === undefined) {
+			continue;
+		}
+
 		let innerValue = value;
 
-		if (_.has(manifest['attributes'][key], 'color')) {
+		// Output color variable from the global variables.
+		if (manifest['attributes'][key]['variable'] === 'color') {
 			innerValue = `var(--global-colors-${innerValue})`;
 		}
 
+		// Output select variable from the options array but dont use value key. It will use variable key.
+		if (_.has(manifest['options'], key) && manifest['attributes'][key]['variable'] === 'select-variable') {
+			const selectVariable = manifest['options'][key].filter((item) => item.value === attributes[key])[0].variable;
+			innerValue = typeof selectVariable === 'undefined' ? attributes[key] : selectVariable;
```

Comment:
Could this be written as `innerValue = selectVariable ?? attributes[key]` ?

Reply:
no sure, but I would leave it like this so it is the same as with PHP for easier debugging.